src/app/lib/__tests__/usePreventChangeOnNumberInputScroll.test.tsx

```diff
@@ -0,0 +1,38 @@
+import { render, fireEvent, act } from '@testing-library/react'
+import { TextInput } from 'grommet/es6/components/TextInput'
+import { usePreventChangeOnNumberInputScroll } from '../usePreventChangeOnNumberInputScroll'
+
+const TextInputFormItem = () => (
+  <TextInput role="input" type="number" {...usePreventChangeOnNumberInputScroll()} />
+)
+
+describe('usePreventChangeOnNumberInputScroll', () => {
+  it('Should focus on input that is already in focus', async () => {
+    const { getByRole } = render(<TextInputFormItem />)
+
+    const numberInput = getByRole('input')
+    fireEvent.change(numberInput, { target: { value: '123' } })
+
+    act(() => {
+      fireEvent.focus(numberInput)
+      fireEvent.wheel(numberInput)
+    })
+
+    // Focuses on body after wheel change, where in actual browser focuses back on the input
+    // expect(numberInput).toHaveFocus()
```

Comment:
This sounds like it should work. Does it just need to wait a bit?

Reply:
Yes, good point.